### ISSUE TITLE
Is it game over?

### ISSUE THREAD

**Issue by JohnMcLear** [joolswills/plugin.video.iplayer#169]
Plugin is flagged as Broken, nothing seems to work...

Did the BBC kill this project?

Tested w/ 2.7.1...

**Comment by joolswills** [comment]
It is for this plugin - at least for now until we switch the backend api - waiting to see what happens with the BBC nitro api.

Info at the kodi forum thread - including some alternative options/plugins

http://forum.kodi.tv/showthread.php?tid=221699&pid=2019016#pid2019016

**Comment by JohnMcLear** [comment]
Wouldn't it normally be good practice to leave an issue open or update the README then?

**Comment by joolswills** [comment]
Yes, but I have been extremely busy - the code only just moved to github too, so I never really got around to finishing the organisation of it before it all went pear shaped. I'll leave the issue open - not sure I actually meant to close it - sorry.
